core/src/main/resources/reference.conf

```diff
@@ -59,11 +59,34 @@ akka {
 
     # Note that without further configuration, all other actors run on the default dispatcher
   }
+
+  coordinated-shutdown.phases {
+    # This phase is used to give time to Cromwell to abort all workflows upon shutdown. It's only used if system.abort-jobs-on-terminate = true
+    abort-all-workflows {
+      # /!\ This depends-on field should NOT be modified
+      depends-on = [service-unbind]
```

Comment:
@Horneth That'd be my preference, granted I wasn't at tech talk today, nor will I be there tomorrow. I do see the general conundrum but I'm afraid down the road this pattern will cause more problems than it solves.

On your second point - yeah. When I first asked that I was just scrolling through the PR and had no idea what was going on :)